logistics/scripts/configuration.packages.json

```diff
@@ -40,6 +40,11 @@
       "PackageVersion": "5.2.47",
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
+    "sk": {
```

Comment:
Should the starter kit package be mentioned at all in here?  Why wouldn't we just manage this version in the starter kit repository directly? @jamessmckay

Reply:
It would probably even make sense to have an equivalent type of this file in the SK repo, even if it only has this one package defined, as long as it was in the SK repo and not mixed in here.